src/main/java/com/smartling/connector/yext/sdk/data/response/folder/ListFolders.java

```diff
@@ -0,0 +1,16 @@
+package com.smartling.connector.yext.sdk.data.response.folder;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class ListFolders
```

Comment:
Maybe `FoldersList`?

Reply:
agree. but other classes already named in such style. didn't what to mix.